### DISCUSSION TITLE
google search result leads to `v3`, not `v4`

### DISCUSSION THREAD

**Discussion by scarf005** [TanStack/query#4868] [General] [answered]
## Problem
| us | korea |
|:-:|:-:|
| ![us](https://user-images.githubusercontent.com/54838975/214495251-9b264833-cbed-4d4e-bf0f-8383d504ebe7.png) | ![korea](https://user-images.githubusercontent.com/54838975/214494644-bd78f529-02cc-4354-8689-0d226b88620e.png) |

first google search result on `react query` points to [`v3`](https://react-query-v3.tanstack.com), not [`v4`](https://tanstack.com/query/latest). v4's not even on first page. in other words, it's almost impossible for newcomers to use `v4` in first try.

## real life issues
1. in my last react query lesson, a large portion of attendants mistakenly used `v3` over `v4` (including me) because above reason.
2. #4385 was caused by OP confusing `v3` with `v4`.

## Solution
`v3` documentations page should point out in header and other noticable places that there's `v4`, the latest version, which is probably what the newcomer was looking for.

**Comment by louis-young** [comment, marked as answer]
Hi 👋 

The think the official name of the library is now "TanStack Query". The official website is first on the SERP for "TanStack Query", and second for "React Query" (second to React Query version 3), although SERPs can differ between people/locations, etc.

I'd be happy to add a "this is not the latest version" header to make this more obvious, what do you think @TkDodo?

**Reply by scarf005** [reply]
i think the confusion comes from that library name on npm is still `@tanstack/react-query`.

**Reply by louis-young** [reply]
Yeah, I definitely understand the conflation. I think the package names include the name of the adapter (React, Solid, Vue, etc). 

I'm more than open to the idea of a warning/informational header, but I can't make that decision. Let's see what Dominik thinks and go from there 🙂
